.travis.yml

```diff
@@ -48,6 +48,7 @@ before_install:
 - sudo apt-get install -y --force-yes r-base-dev=3.1.3-1trusty
 - sudo apt-get install -y --force-yes r-base-core=3.1.3-1trusty
 - sudo Rscript scripts/install_R_packages.R
+- sudo Rscript scripts/install_R_packages_for_testing.R
```

Comment:
There is documentation (on the forum and in README) that will need to be updated to match this step. 

Also, you should update the Dockerfile.